CHANGES.rst

```diff
@@ -1,6 +1,23 @@
 Changes
 =======
 
+Version 0.5.0 (2019-04-23)
+--------------------------
+
+- Enables to create workflows on demand.
+- Add support for interactive sessions to manage workflows directly with interactive tools such as Jupyter.
+- Adds REANA shared storage to interactive sessions for easier viewing of the results.
```

Comment:
I think this would be already included in the previous bullet point, since we mention that the workspace can be accessed by the interactive session.